packages/config/config/devices/0x011a/zwn-bpc_0.0-5.9.json

```diff
@@ -1,39 +0,0 @@
-{
-	"manufacturer": "Enerwave / Wenzhou MTLC Electric Appliances Co., Ltd.",
-	"manufacturerId": "0x011a",
-	"label": "ZWN-BPC",
-	"description": "PIR Sensor",
-	"devices": [
-		{
-			"productType": "0x0601",
-			"productId": "0x0901"
-		}
-	],
-	"firmwareVersion": {
-		"min": "0.0",
-		"max": "5.9"
-	},
-	"associations": {
-		"1": {
-			"label": "Reports",
-			"maxNodes": 5,
-			"isLifeline": true
-		}
-	},
-	"paramInformation": [
-		{
-			"#": "0",
```

Comment:
Another param number change 0->1.
Not sure if the old one was correct, but there is another variant of the ZWN-BPC <= 5.9 which also uses number 0.